README.md

```diff
@@ -52,7 +52,7 @@ The base environment most likely will not work.
 Install Casanovo as a Python package from this repository (requires 3.8 ≥ [Python version] > 3.10 , dependencies will be installed automatically as needed):
```

Comment:
**question:** Does Python (fully) 3.10 work? The instructions currently dictate Python 3.8 or 3.9, whereas the tests Action uses Python 3.10.

Reply:
I actually don't know. Do you know why it didn't work before? My thought is we may have tried it shortly after 3.10 was released, before most packages had a chance to update.

Reply:
I haven't tested it with 3.10 and just followed the existing description in the README, so I don't know.

Reply:
I haven't run into trouble with 3.10 yet, but maybe we should wait to change the README until more tests are added.